modules/panorama/main.tf

```diff
@@ -6,6 +6,7 @@ resource "azurerm_public_ip" "this" {
   location            = var.location
   resource_group_name = var.resource_group_name
   allocation_method   = "Static"
+  sku                 = "Standard"
```

Comment:
Good catch, Azure says "Use Standard SKU public IP for Availability Zone scenarios".